src/plone/restapi/tests/test_atfield_serializer.py

```diff
@@ -70,12 +73,18 @@ def test_lines_field_serialization_returns_list(self):
         self.assertTrue(isinstance(value, list), 'Not a <list>')
         self.assertEqual([u'foo', u'bar'], value)
 
-    def test_file_field_serialization_returns_unicode(self):
+    def test_file_field_serialization_returns_dict(self):
         value = self.serialize('testFileField', 'spam and eggs',
-                               filename='spam.txt', mimetype='text/plain')
-        self.assertTrue(isinstance(value, unicode), 'Not an <unicode>')
-        self.assertEqual(u'http://nohost/plone/doc1/@@download/testFileField',
-                         value)
+                               mimetype='text/plain')
+        self.assertTrue(isinstance(value, dict), 'Not a <dict>')
+
+        url = u'http://nohost/plone/doc1/@@download/testFileField'
+        self.assertEqual(
+            {u'filename': '',
```

Comment:
@erral thank you, that works. Though after looking at it again, it also seems to work for that field type by passing a [keyword argument to the mutator](https://github.com/plone/plone.restapi/pull/115/files/1d49315bd5533d6640a7f536a406ea76b800a8f6#diff-06974b9abd48f281685ec4494e725f08R28) - which I somehow couldn't get to work at first for that field type.